package.json

```diff
@@ -10,6 +10,7 @@
     "client:clear": "rimraf ./build/static",
     "client:watch": "cross-env NODE_ENV=development webpack --mode development --watch --progress",
     "db:migrate": "sequelize db:migrate",
+    "db:new-migration": "./node_modules/.bin/sequelize-cli migration:generate --name",
```

Comment:
You can skip the relative reference to the binary and just write `sequelize migration:generate --name`

Reply:
tbh that actually doesn't work for me, though maybe I've set the project up wrong - I need to at least use npx

Reply:
Ok, interesting, I guess thats a setup problem indeed. Do you have a globally installed sequelize version conflicting?

Reply:
Don't think so - I just get `sequelize: command not found`